packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -118,8 +121,20 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
         taskViewService.removeTask(task);
         notificationService.showSuccessfulExecution(executionName);
 
+        if (data!.toString()!.match(SERVER_INFO_REGEX)) {
```

Comment:
This is the crux of the PR. We retrieve the URL info from server response and use it instead of the hardcoded localhost:3333.

Reply:
Would devServerService be a better place for a method like this?

Reply:
Sure! Like data.toString() to devServerService and let it parsing and setting the Url?

Reply:
yeah maybe something like setBaseUrlFromDevServerUpMessage

Reply:
easier to have it unit tested too